src/payment/strategies/braintree/braintree-sdk-creator.ts

```diff
@@ -43,19 +43,23 @@ export default class BraintreeSDKCreator {
                 this.getClient(),
                 this._braintreeScriptLoader.loadPaypal(),
             ])
-            .then(([client, paypal]) => paypal.create({ client }));
+                .then(([client, paypal]) => paypal.create({ client }));
         }
 
         return this._paypal;
     }
 
-    getPaypalCheckout(): Promise<BraintreePaypalCheckout> {
+    getPaypalCheckout(renderButtonCallback: any, context: any): Promise<BraintreePaypalCheckout> {
```

Comment:
you should have proper types for these arguments. Using `any` defeats the purpose of using typescript

Reply:
fixed